src/npctalk.cpp

```diff
@@ -2054,6 +2054,16 @@ void parse_tags( std::string &phrase, const talker &u, const talker &me, const d
     do {
         fa = phrase.find( '<' );
         fb = phrase.find( '>' );
+        // Skip the <color_XXX> tag
+        if( fa != std::string::npos && phrase.compare( fa + 1, 6, "color_" ) == 0 ) {
+            fa = phrase.find( '<', fa + 7 );
+            fb = phrase.find( '>', fa );
+        }
+        // Skip the </color> tag
+        if( fa != std::string::npos && phrase.compare( fa + 1, 7, "/color>" ) == 0 ) {
+            fa = phrase.find( '<', fa + 8 );
+            fb = phrase.find( '>', fa );
+        }
```

Comment:
I think this will handle consecutive color tags (for example `<color_xxx><color_xxx>`) incorrectly.

Reply:
I understand. Here, a loop should be used to exclude consecutive color tags.